sqlitecluster/SQLiteNode.cpp

```diff
@@ -2700,14 +2757,6 @@ void SQLiteNode::_sendStandupResponse(SQLitePeer* peer, const SData& message) {
         return;
     }
 
-    if (_forkedFrom.count(peer->name)) {
-        PHMMM("Forked from peer, can't approve standup.");
-        response["Response"] = "abstain";
```

Comment:
Am I understanding correctly that here we don't `deny` the STANDUP request because we have no way of checking that we forked from a specific peer? What would happen in this case if `forked = true`?

Reply:
We used to require all connected peers to respond before we'd go leading. This was cautionary - we didn't want to stand up with 3 peers approving and have the 4th one respond last saying "STOP! NO!" so we waited for all peers to respond.

A forked peer's only reasonable response to STANDUP used to be "DENY" because it would notice that the peer standing up had (from its perspective) invalid DB hashes. However, this would mean peers that were forked from one another would always stop a cluster from forming. We added ABSTAIN to allow it not to vote if the DB hashes didn't match, but still have every connected peer respond.

We have since removed the requirement for all peers to respond to go LEADING, we can now go LEADING as soon as enough peers to form quorum approve, so ABSTAIN is no longer really useful.